launchable/commands/record/build.py

```diff
@@ -172,9 +176,19 @@ def build(ctx: click.core.Context, build_name: str, source: List[str], max_days:
 
         payload = {
             "buildNumber": build_name,
-            "commitHashes": commitHashes
+            "commitHashes": commitHashes,
         }
 
+        _links = capture_link(os.environ)
+        if len(links) != 0:
+            for link in normalize_key_value_types(links):
+                _links.append({
+                    "title": link[0],
+                    "url": link[1],
+                    "kind": "CUSTOM_LINK"
```

Comment:
[nits]
Probably better to use `LinkKind.CUSTOM_LINK`.
https://github.com/launchableinc/cli/pull/529/files#diff-280a9bc4f9d9c38c29d666a70adf8c4e216a1abb22f548b3a011f9ac00b6bf10R18